docs/cookbook/store.md

```diff
@@ -197,16 +197,12 @@ Alternatively, you can provide your own types at the point of use.
 <script lang="ts">
 
 import { Component, Vue } from 'nuxt-property-decorator'
-import { getters, RootState } from '~/store'
+import { RootState } from '~/store'
 
 @Component
 export default class MyComponent extends Vue {
     get myThings() {
         return (this.$store.state as RootState).things
     }
-
-    mounted() {
-        const name = this.$store.getters['name'] as ReturnType<typeof getters.name>
```

Comment:
I think the point here was to show how to `provide own types at the point of use`, why should we remove it then ?

Reply:
Oh, I think updated the original code instead the spanish code 😅

I will fix that.